templates/parachain/README.md

```diff
@@ -197,6 +196,40 @@ Development parachains:
 - 💰 Are preconfigured with a genesis state that includes several prefunded development accounts.
 - 🧑‍⚖️ Development accounts are used as validators, collators, and `sudo` accounts.
 
+## Runtime development
+
+We recommend using [`chopsticks`](https://github.com/AcalaNetwork/chopsticks) when the focus is more on the runtime
+development and `OmniNode` is enough as is.
+
+### Install chopsticks
+
+To use `chopsticks`, please install the latest version according to the installation [guide](https://github.com/AcalaNetwork/chopsticks?tab=readme-ov-file#install).
+
+### Build a raw chain spec
+
+Build the `parachain-template-runtime` as mentioned before in this guide and use `chain-spec-builder`
+again but this time by passing `--raw-storage` flag:
+
+```sh
+chain-spec-builder create --raw-storage --relay-chain "rococo-local" --para-id 1000 --runtime \
+    target/release/wbuild/parachain-template-runtime/parachain_template_runtime.wasm named-preset development
+```
+
+### Start `chopsticks` with the chain spec
+
+We'll also need to set `--allow-unresolved-imports=true` because the `parachain-template-runtime` is not built
```

Comment:
From what I understand it is the otherway round. You tried to use chopsticks earlier with a runtime which was built using `--features runtime-benchmarks`. This makes the runtime require additional host functions. You are telling chopsticks to ignore these hostfunctions by using this flag.

So if users build their runtimes without `--features runtime-benchmarks` they should not require this flag.

Reply:
updated the readme here: [f696909](https://github.com/paritytech/polkadot-sdk/pull/6646/commits/f6969094b6af59df36d0fb90c61d51609f7f58bd)